app/views/transport_requisitions/index.html.erb

```diff
@@ -0,0 +1,116 @@
+<div class="row wrapper border-bottom white-bg page-heading">
+  <div class="col-sm-4">
+    <h2>Transport Requisitions</h2>
+    <ol class="breadcrumb">
+      <li>
+        <%= link_to 'Home', '/' %>
+      </li>
+      <li class="active">
+        <strong>Transport Requisitions</strong>
+      </li>
+    </ol>
+  </div>
+  <div class="col-sm-8">
+    <div class="title-action">
+      <a class="btn btn-primary" data-target="#new-transport-requisition" data-toggle="modal">
+          New Transport Requisition
+      </a>
+    </div>
+  </div>
+</div>
+
+
+<div class="wrapper wrapper-content">
+  <div class="ibox float-e-margins">
+    <div class="row">
+      <div class="ibox-content col-md-12 col-sm-12 col-xs-12">
+        <div class="col-md-6 col-sm-6 col-xs-6" id="tr-operation-filter">
+          <label class='control-label'>Operation</label>
+          <%= select_tag :operation, options_from_collection_for_select(Operation.all.order(:name), 'id', 'name', params[:operation_id]), {:required => true,:class=> 'data-parsley-required form-control width-300px'} %>
+        </div>
+
+      </div>
+    </div>
+  </div>
+  <div class="ibox float-e-margins">
+    <div class="ibox-content">
+
+      <table class="table table-striped cats-datatable">
+        <thead>
+          <tr>
+            <th>Reference number</th>
+            <th>Region</th>
+            <th>No of Destinations</th>
+            <th>Total Qty.</th>
+            <th>Created date</th>
+            <th>Status</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          <% @transport_requisitions.each do |transport_requisition| %>
+            <tr>
+              <td>
+              <%= link_to transport_requisition.reference_number, transport_requisition_url('en',transport_requisition) %>
+              <td><%= Location.find(transport_requisition&.location_id)&.name %></td>
```

Comment:
@nathnael let's try to load everything the view needs in the controller; through includes so that we don't have to write this sort of logic in the view. Difficult to trace problems, debug and trace.